packages/manager/apps/pci-public-ip/src/api/hooks/useFloatingIP.ts

```diff
@@ -27,6 +28,9 @@ const aggregateFloatingIPs = (
 
       floatingIPResult.associatedEntity.name = instance ? instance.name : '';
     }
+    floatingIPResult.associatedEntityId = floatingIP.associatedEntity.id;
```

Comment:
you already have the id in **associatedEntity** object ! why you set it again ?

Reply:
the function `applyFilter` doesn't give the possibility de filter a nested property ( in this case the property `id` of `associatedEntity`, i need to get two specific property to give possibility to filter